api_app/analyzers_manager/migrations/0067_analyzer_config_feodo_tracker.py

```diff
@@ -0,0 +1,195 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": {
+            "minute": "0",
+            "hour": "0",
+            "day_of_week": "*",
+            "day_of_month": "*",
+            "month_of_year": "*",
+        },
+        "update_task": {
+            "crontab": {
+                "minute": "0",
+                "hour": "0",
+                "day_of_week": "*",
+                "day_of_month": "*",
+                "month_of_year": "*",
+            },
+            "name": "api_app.analyzers_manager.observable_analyzers.feodo_tracker.Feodo_TrackerUpdate",
+            "task": "intel_owl.tasks.update",
+            "kwargs": '{"python_module_pk": 151}',
```

Comment:
@0ssigeno is there a way to avoid to have hardcoded this pk, leading to possible conflicts?

Reply:
Ok, this requires a change in the backend that changes the `python_module_pk` to something that is invariant on the number of python modules that you have in your db.

@g4ze, the pr is good, I will merge this once i have made some changes in the backend to correctly support the update function